### DISCUSSION TITLE
zero pcb with recommended RGB-TFT and CRT logo problem

### DISCUSSION THREAD

**Discussion by StmLord** [frntc/Sidekick64#64] [General]
I thought to create logos for my CRT collection but I have a weird problem.
Logos are X/Y 240/240 pixels TGA files, I put them in the same directory where the CRTs are.
But the RGB-TFT display them with inverted (only) the Y dimension!
All embedded logos like menu, music, etc are displayed properly.
Also photoshop tga saved files do not work at all, so I had to use an online tga converter, but on my computer are showing correctly.

**Comment by frntc** [comment]
In fact TGA has a flag in the file header for Y-mirroring -- which is blatantly ignored :-) I decided to not change this as all existing logos might then be flipped. The solution simply is to flip the logos you create (irfanview, e.g. can do that in a batch process). Photoshop iirc is a bit picky, your might have to create an alpha channel and pay attention to the export settings (but it does work).
